packages/wellknown-did-verifier/src/types/IWellKnownDidVerifier.ts

```diff
@@ -24,12 +20,12 @@ export interface IWellKnownDidVerifierOptionsArgs {
 }
 
 export interface IRegisterSignatureVerificationArgs {
-  key: string
+  signatureVerificationKey: string
```

Comment:
callbackName might be better, because this can still be confused with a public key for signature verification